dashboard/app/views/levels/editors/_all.html.haml

```diff
@@ -22,11 +22,26 @@
       Notes for yourself or other Levelbuilders. Viewable here and here alone.
     = f.text_area :notes, rows: 6, class: "input-block-level"
 
-.field
-  = f.label 'bubble_choice_description'
-  %p
-    Description of this level and why a student would want to choose it as part of a BubbleChoice progression.
-  = f.text_area :bubble_choice_description, rows: 3, class: "input-block-level"
+
+- if !@level.is_a?(BubbleChoice)
+  %legend.control-legend.collapsed{data: {toggle: "collapse", target: "#bubble_choice"}}
+    Bubble Choice
+  #bubble_choice.in.collapse
+    - unless @level.is_a?(DSLDefined) || @level.is_a?(Unplugged)
+      .field
+        = f.label :display_name
+        %p Set the name that will show for this level on the parent bubble choice level. Also will override how this level is displayed on the Course Overview page.
+        = f.text_field :display_name, placeholder: '(defaults to Name)'
```

Comment:
And script overview? I'm still fuzzy on how display_name is used/not.

Reply:
On the script overview page the display name will show up in certain cases.... we really want to move away from it for most cases but bubble choice needs something like this to display a name for the sublevel

Reply:
and what it means for something to be deprecated in this context

Reply:
Deprecated just means we moved it to the bottom of the level edit page in a collapsed area in order to see if in the future we could remove it